### DISCUSSION TITLE
Deploy SPA build on GitHub Pages

### DISCUSSION THREAD

**Discussion by xeoneux** [remix-run/remix#8523] [Q&A]
How would one go about deploying a SPA build on GH Pages with GitHub Actions?

**Comment by brophdawg11** [comment]
Github pages just serves from an `index.html` file in your configured branch, so you'll need to set that up on your end.  `remix build` will generate the `index.html` file in `build/client/index.html` for a Remix SPA, so you may need to move that and any potential assets into the right locations for GH pages to serve it (probably via a github actions workflow).

This guide should be helpful!  https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site
